src/interfaces/opty/contracts/IRegistry.sol

```diff
@@ -248,13 +248,13 @@ interface IRegistry {
     /**
      * @notice Maps the Vault contract with underlying assets and riskProfile
      * @param _vault Vault contract address
-     * @param _riskProfile Risk profile mapped to the vault contract
+     * @param _riskProfileCode Risk profile mapped to the vault contract
```

Comment:
- [ ] Registry and vault interfaces should be removed from this package.

Reply:
Done